R/server_manager.R

```diff
@@ -291,7 +298,12 @@ vault_install <- function(dest, quiet, version) {
     tmp <- tempfile()
     dir_create(tmp)
     utils::unzip(zip, exdir = tmp)
-    ok <- file.copy(file.path(tmp, "vault"), dest_bin)
+
+    ok <- file.copy(file.path(tmp, vault_exe_filename()), dest_bin)
+    if (!ok) {
+      warning(sprintf("Failed to copy vault to %s", dest_bin))
```

Comment:
If you want to check a successful copy, please see `file_copy` in orderly (copy it and the tests here)

Reply:
Done - added file_copy to R/util.R, with tests in tests/testthat/test-util.R, and modified the above to call it. (The above is the only place where file.copy was called)